blenderproc/python/utility/Initializer.py

```diff
@@ -37,9 +38,28 @@ def init(horizon_color: list = [0.05, 0.05, 0.05], compute_device: str = "GPU",
     bpy.context.scene.render.engine = 'CYCLES'
 
     if platform == "darwin" or compute_device == "CPU":
-        # if there is no gpu support (mac os) or if cpu is specified as compute device, then use the cpu with maximum power
-        bpy.context.scene.cycles.device = "CPU"
-        bpy.context.scene.render.threads = multiprocessing.cpu_count()
+        import platform as platform_locally
+        mac_version = platform_locally.mac_ver()[0]
```

Comment:
This probably will no work when the platform is linux and only compute_device was set to cpu

Reply:
Good catch, I fixed it.